include/spirv-tools/optimizer.hpp

```diff
@@ -101,10 +101,17 @@ class Optimizer {
   // from time to time.
   Optimizer& RegisterSizePasses();
 
-  // Registers passes that have been prescribed for WebGPU environments.
+  // Registers passes that have been prescribed for converting from Vulkan to
+  // WebGPU.
   // This sequence of passes is subject to constant review and will change
   // from time to time.
```

Comment:
Could you format the comments better?  The line breaks look awkward.

Reply:
Done